geode/src/test/java/org/apache/calcite/adapter/geode/rel/GeodeBookstoreTest.java

```diff
@@ -379,10 +385,36 @@ public void testSelectWithNestedPdx2() {
     calciteAssert()
         .query("select primaryAddress from geode.BookCustomer limit 2")
         .returnsCount(2)
-        .returns("primaryAddress=PDX[addressLine1,addressLine2,addressLine3,city,state,"
-            + "postalCode,country,phoneNumber,addressTag]\n"
-            + "primaryAddress=PDX[addressLine1,addressLine2,addressLine3,city,state,postalCode,"
-            + "country,phoneNumber,addressTag]\n")
+        .returns(new Consumer<ResultSet>() {
```

Comment:
I update this case to make it pass.
But I'm not quite familiar with the logic, hopes someone more familiar with it can review this.